src/current/v24.1/molt-fetch.md

```diff
@@ -91,9 +89,34 @@ Complete the following items before using MOLT Fetch:
 			~~~
 
 		- Ensure the Google Cloud Storage bucket is created and accessible to CockroachDB.
-	
+
+## Best practices
+
+- To prevent connections from prematurely terminating during data export, set the following to high values on the source database:
```

Comment:
```suggestion
- To prevent connections from terminating prematurely during data export, set the following to high values on the source database:
```
Nit to avoid splitting the infinitive, but I think it reads more clearly too